rocAL/rocAL/source/api/rocal_api_info.cpp

```diff
@@ -100,6 +100,23 @@ rocalGetRemainingImages(RocalContext p_context)
     return count;
 }
 
+size_t  ROCAL_API_CALL
+rocalGetLastBatchPaddedSize(RocalContext p_context)
+{
+    auto context = static_cast<Context *>(p_context);
+    size_t count = 0;
+    try
+    {
+        count = context->master_graph->last_batch_size();
```

Comment:
Should the api be renamed as last_batch_padded_size() instead of last_batch_size()
Because the outer api name is rocalGetLastBatchPaddedSize

Reply:
Renamed it.